hid-uclogic-rdescs.h

```diff
@@ -0,0 +1,76 @@
+#ifndef UCLOGIC_REPORT_DESCRIPTORS
+#define UCLOGIC_REPORT_DESCRIPTORS
```

Comment:
Could you please name this macro after the file, with underscore in front? I.e. `_HID_UCLOGIC_RDESCS_H`?